formats/filtered/src/Filtered.php

```diff
@@ -164,6 +165,7 @@ protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		$result = [];
 		while ( $row = $res->getNext() ) {
 			$result[ $this->uniqid() ] = new ResultItem( $row, $this );
+			usleep( 1 );
```

Comment:
I'm a bit wary about such arbitrary stop process as depends on an environment and may cause non-deterministic behaviour.